### DISCUSSION TITLE
Filament shield plugin is not working in filament multi tenancy

### DISCUSSION THREAD

**Discussion by Aung-Zaw-Phyo** [filamentphp/filament#13485] [Help]
### Package

Other

### Package Version

3

### How can we help you?

How can i implement shield plugin in multi tenancy

**Comment by leandrocfe** [comment]
Please ask in the plugin github discussions or discord channel
https://filamentphp.com/plugins/bezhansalleh-shield

**Comment by micraux** [comment]
Hello,

I tried Shield, but encountered one problem: if a User belongs to many Teams, he will always has the same role.

For example, the User can't be an "Author" in Team A and a "Contributor" in Team B.

I like Shield a lot but my project needed this functionality, so I switched to [another plugin](https://filamentphp.com/plugins/tharinda-rodrigo-spatie-roles-permissions)  (also based on Spatie Permission).

You might want to have a look...
